### DISCUSSION TITLE
Character Encoding of CSV file export

### DISCUSSION THREAD

**Discussion by xnohat** [Budibase/budibase#5453] [Help]
Hi team,

I got an issue when export a table to CSV.
I using Vietnamese language texts in table, text character display correctly on browser
But export this table to CSV and open in Microsoft Excel or any Text Editor with UTF-8 Character Encoding, all character of Vietnamese language corrupt (Vietnamese have accent characters like á à ổ ơ í ê ... beside latin a-z)
![image](https://user-images.githubusercontent.com/4613808/163685846-a1599c0d-a24c-4d94-97d1-a1786a444815.png)

Can anyone explain to me what is character encoding Budibase using when export table data to csv ?

I attached my test app with Vietnamese language here
[test export-export-1650131273202.txt](https://github.com/Budibase/budibase/files/8500405/test.export-export-1650131273202.txt)

Thanks in advance,

**Comment by melohagan** [comment]
Hi @xnohat 

I believe this is a related issue: https://github.com/Budibase/budibase/issues/3844

**Reply by xnohat** [reply]
thanks, that same issue
